tests/MulticastUDPClient.py

```
@@ -111,68 +111,19 @@
 		raise ModuleNotFoundError(
 			"[CWE-440] OMG! sys.modules is not available or empty."
 		) from None
-except ImportError as err:
-	raise ImportError("[CWE-440] Unable to import sys module.") from err
+except ImportError as baton:
+	raise ImportError("[CWE-440] Unable to import sys module.") from baton
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)